be/src/olap/snapshot_manager.h

```diff
@@ -70,6 +70,8 @@ class SnapshotManager {
                 
     static SnapshotManager* instance();
 
+    OLAPStatus convert_rowset_ids(DataDir& data_dir, const string& clone_dir, int64_t tablet_id, const int32_t& schema_hash);
```

Comment:
int64_t tablet_id, const int32_t schema_hash?

Reply:
yes， schema hash is int32